apps/desktop-wallet/src/modals/NFTDetailsModal.tsx

```diff
@@ -96,3 +131,8 @@ const NFTMetadataContainer = styled.div`
   flex-direction: column;
   gap: var(--spacing-3);
 `
+
+const matchesNFTCollectionUriMetaDataSchema = (nftCollection: NFTCollectionUriMetaData) =>
+  typeof nftCollection.name === 'string' &&
+  typeof nftCollection.image === 'string' &&
+  typeof nftCollection.description === 'string'
```

Comment:
Same as above. It will be moved to shared when we extend the Tanstack integration.